tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerTransactionGraphTest.java

```diff
@@ -429,6 +445,7 @@ public void shouldRemoveIndexForRemovedVertex() throws InterruptedException {
      }
 
     @Test
+    @FeatureRequirement(featureClass = Graph.Features.GraphFeatures.class, feature = FEATURE_USER_SUPPLIED_IDS)
```

Comment:
Are all of these `@FeatureRequirement` annotations needed in TinkerTransactionGraphTest or are those annotations just for stuff under `gremlin-test`?

Reply:
Yes I was asked to add this tag to all tests that have user supplied vertex ids but I'll need Ken to double check whether I hit them all or too many